### PR TITLE
changed activations to _activations

### PR DESCRIPTION
**Pull request by priyatharsan**

### CODE REVIEW

**Comment by priyatharsan** [comment]
the model is not saving although. Getting `OSError`
capture

**Comment by rfratila** [comment]
Could you print model1 from the example script? I would like to see what the output looks like @priyatharsan

**Comment by priyatharsan** [comment]
capture

**Comment by priyatharsan** [comment]
Now it prints good enough 
capture

**Comment by nirtiac** [comment]
@priyatharsan ok I somewhat expected that for the save function for windows... I've never used a windows machine so I don't know how it works. if it's an easy fix can you push it? if not I'll do some research

**Comment by nirtiac** [comment]
@priyatharsan does the file name change just pushed help?

**Comment by priyatharsan** [comment]
`saved_path` is now fixed, but I get "Python not responding" popup error during runtime of saving, but it gets saved.

File: examples/fashion_conv_dense_test.py [hunk 1]
```diff
@@ -59,6 +59,7 @@
                         initializer=None,
                         bias_init=None, # None or value
                         norm=None,
+                        pool_size=None,
```
Comment (priyatharsan) [inline examples/fashion_conv_dense_test.py]: good catch! I missed it

**Pull request merged**